### PR TITLE
Prod Squad Norms: Add updates from discussion

### PR DESCRIPTION
**Pull request by NickHeiner**

### CODE REVIEW

**Comment by NickHeiner** [comment]
@aroltsch @kierachell can you add the links to the docs you mentioned?

File: docs/prod-squad.md [hunk 1]
```diff
@@ -59,6 +63,7 @@ If you are on prod squad, you must have access to the following:
 1. Sentry
 1. AWS
 1. Production (i.e. you can log into prod as a user)
+    * Note: we are currently in the process of getting production access. It's ok if you don't have this yet.
 1. SSH into production boxes
```
Comment (kierachell) [inline docs/prod-squad.md]: https://github.com/department-of-veterans-affairs/appeals-deployment/blob/master/docs/ssh-user.md

**Comment by kierachell** [comment]
- Logs: https://github.com/department-of-veterans-affairs/appeals-deployment/blob/master/docs/cheatsheet/logs.md

- SSH Access: https://github.com/department-of-veterans-affairs/appeals-deployment/blob/master/docs/ssh-user.md

-

**Comment by mmlumba** [comment]
It would be helpful to add a link to how post mortems are written: https://github.com/department-of-veterans-affairs/caseflow/issues?q=is%3Aissue+label%3Apost-mortem

**Pull request merged**